PennyPincher.cs

```diff
@@ -74,173 +78,118 @@ public PennyPincher(DalamudPluginInterface pluginInterface, ChatGui chat, Comman
 
         public void Dispose()
         {
-            this.gn.NetworkMessage -= this.OnNetworkEvent;
-            this.c.RemoveHandler(commandName);
-            this.pi.Dispose();
+            GameNetwork.NetworkMessage -= OnNetworkEvent;
+            PluginInterface.UiBuilder.Draw -= DrawWindow;
+            PluginInterface.UiBuilder.OpenConfigUi -= OpenConfigUi;
+            CommandManager.RemoveHandler(commandName);
+            PluginInterface.Dispose();
         }
-
-        private void PrintSetting(string name, bool setting)
+        
+        private void Command(string command, string arguments)
         {
-            this.chat.Print(name + (setting ? " enabled." : " disabled."));
+            _config = true;
+        }
+        
+        private void OpenConfigUi()
+        {
+            _config = true;
         }
 
-        private void OnCommand(string command, string args)
+        private void DrawWindow()
         {
-            if (args == string.Empty)
+            if (!_config) return;
+            
+            ImGui.SetNextWindowSize(new Num.Vector2(600, 600), ImGuiCond.FirstUseEver);
+            ImGui.Begin($"{Name} Config", ref _config);
+            
+            ImGui.InputInt("Minimum Price", ref configMin);
+            ImGui.TextWrapped("Sets a minimum value to be copied. <min> cannot be below 1.");
+            
+            ImGui.InputInt("Mod", ref configMod);
+            ImGui.TextWrapped("Adjusts base price by subtracting <price> % <mod> from <price> before subtracting <delta>.\nThis makes the last digits of your posted prices consistent.");
+            
+            ImGui.InputInt("Delta", ref configDelta);
+            ImGui.TextWrapped("Sets the undercutting amount to be <delta>.");
```

Comment:
Can you reorder this so that's delta, min, mod? I think delta is most important and should be at the top.

Reply:
> 
> 
> Can you reorder this so that's delta, min, mod? I think delta is most important and should be at the top.
Done!